### ISSUE TITLE
changesNotSentForReview

### ISSUE THREAD

**Issue by romatallinn** [r0adkll/upload-google-play#132]
**Describe the bug**
The flow started failing with:
```
Error: Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.
```

But when I do as it says, I am also getting a warning:
```
Warning: Unexpected input(s) 'changesNotSentForReview', valid inputs are ['serviceAccountJson', 'serviceAccountJsonPlainText', 'packageName', 'releaseFile', 'releaseFiles', 'releaseName', 'track', 'inAppUpdatePriority', 'userFraction', 'status', 'whatsNewDirectory', 'mappingFile']
```

<img width="962" alt="Screenshot 2022-07-27 at 14 39 47" src="https://user-images.githubusercontent.com/14923964/181249607-a10a17cb-ef51-458a-98c0-da36795e156c.png">

**Workflow Step Configuration**

```yml
      - name: Publishing app to Play Store
        uses: r0adkll/upload-google-play@v1
        env:
          FLUTTER_BUILD_NAME: ${{ github.event.inputs.FLUTTER_BUILD_NAME }}
          FLUTTER_BUILD_NUMBER: ${{ github.event.inputs.FLUTTER_BUILD_NUMBER }}
        with:
          releaseName: "v${{ env.FLUTTER_BUILD_NAME }}+${{ env.FLUTTER_BUILD_NUMBER }}"
          serviceAccountJsonPlainText: ${{ secrets.FS_ANDROID_SERVICE_ACCOUNT_JSON }}
          packageName: ****
          releaseFiles: build/app/outputs/bundle/release/app-release.aab
          changesNotSentForReview: true
          status: draft
          track: beta
```

**Comment by boswelja** [comment]
The issue was caused by `changesNotSentForReview` not being available in the latest release. It is now available in `v1.0.18`

**Comment by tk-nguyen** [comment]
I still encounter this error after the latest update. The issue is the same as above.
